distro/exporter.go

```diff
@@ -298,3 +305,27 @@ func noneEnvVarSet(envs ...string) bool {
 func notNone(s string) bool {
 	return s != "" && s != "none"
 }
+
+func getOTLPProtocol(l logr.Logger, signalKey string) string {
+	// signal-specific key takes precedence
+	if v := os.Getenv(signalKey); v != "" {
+		if v == defaultOTLPProtocol || v == httpProtobuf {
```

Comment:
this would be more readable

```suggestion
		if v == otlpProtocolGRPC || v == otlpProtocolHTTPProtobuf {
```
and

```go
const defaultOTLPProtocol = otlpProtocolGRPC
```

Reply:
Fixed in 7d9b5a5